main.go

```diff
@@ -199,6 +200,8 @@ func main() {
 	spaceAuthzService := authz.NewAuthzService(config)
 	service.Use(authz.InjectAuthzService(spaceAuthzService))
 
+	service.Use(metric.Recorder())
```

Comment:
metric recorder is added as last handler (goa-middleware) before actual handler get call.

Below is the **new call stack** after adding this handler:
/**metric.Recorder**.func1.1 (/home/nvirani/go/src//metric/metric_recorder.go:16)
/space/**authz.InjectAuthzService**.func1.1 (/home/nvirani/go/src//space/authz/authz.go:112)
/log.LogRequest.func1.1 (/home/nvirani/go/src//log/log_request.go:86)
/login.InjectTokenManager.func1.1 (/home/nvirani/go/src//login/service.go:244)
/goamiddleware.TokenContext.func1.1 (/home/nvirani/go/src//goamiddleware/jwt_token_context.go:75)
/vendor/github.com/goadesign/goa/middleware.Recover.func1.1 (/home/nvirani/go/src//vendor/github.com/goadesign/goa/middleware/recover.go:37)
/jsonapi.ErrorHandler.func1.1 (/home/nvirani/go/src//jsonapi/error_handler.go:38)
/vendor/github.com/goadesign/goa/middleware/gzip.Middleware.func2.1 (/home/nvirani/go/src//vendor/github.com/goadesign/goa/middleware/gzip/middleware.go:98)
/vendor/github.com/goadesign/goa/middleware.RequestIDWithHeaderAndLengthLimit.func1.1 (/home/nvirani/go/src//vendor/github.com/goadesign/goa/middleware/request_id.go:63)